Still one failure:

```
  FAILED kubernetes_tests/test_kubernetes_pod_operator.py::TestKubernetesPodOperatorSystem::test_pod_template_file
```

https://github.com/apache/airflow/pull/15507/checks?check_run_id=2424175787#step:8:3442

--- a/kubernetes_tests/test_kubernetes_pod_operator.py
+++ b/kubernetes_tests/test_kubernetes_pod_operator.py
@@ -876,6 +876,7 @@ class TestKubernetesPodOperatorSystem(unittest.TestCase):
                 'hostNetwork': False,
                 'imagePullSecrets': [],
                 'initContainers': [],
+                'nodeSelector': {},
                 'restartPolicy': 'Never',
                 'securityContext': {},
                 'serviceAccountName': 'default',

